docs/components/input.md

```diff
@@ -306,6 +306,25 @@ const App = () => (
 );
 ```
 
+### Currency
+
+Has the `prefix` of `$` and `suffix` of `USD`. It only allows numeric input. It converts numbers into comma separated currency format.
```

Comment:
Would be good to mention the attribute like the other sections
> Use the `currency` attribute to set the currency symbol and code. It only allows integers which will be converted into a comma-separated currency format.
Only `usd` is implemented.

Reply:
I see that you're setting `currency` as an attribute with a string value rather than a boolean like `pill`, which would make sense if we're planning to support currencies other than `usd`. If that's the case, the we should go ahead and pull the prefix and suffix strings from an enum so we can easily add other currency types later. (One use case I can think of that's plausible in the near future is Canadian currency).